### PR TITLE
Three windings transformer model changes

### PR DESCRIPTION
**Pull request by zamarrenolm**
Signed-off-by: José Antonio Marqués <marquesja@aia.es>

This pull request contains the updated documentation linked to the change in the three windings transformer prepared for powsybl-core version 3.1.0.

### CODE REVIEW

File: docs/iidm/model/threeWindingsTransformer.md [hunk 1]
```diff
@@ -5,23 +5,34 @@ layout: default
 
 The `com.powsybl.iidm.network.ThreeWindingsTransformer` interface is used to model a three windings power transformer.
 A three windings power transformer is connected to three voltage levels (side 1, side 2 and side 3) that belong to the
-same substation:
+same substation, usually:
```
Comment (MioRtia) [inline docs/iidm/model/threeWindingsTransformer.md]: Why "usually"? Is it possible? Should it be indicated, keeping in mind it is forbidden in IIDM and we are describing the IIDM model?

File: docs/iidm/model/threeWindingsTransformer.md [hunk 2]
```diff
@@ -5,23 +5,34 @@ layout: default
...
 - Side 1 is the primary side (high voltage)
-- Side 2 and Side 3 can indifferently be the secondary side (medium voltage) or the tertiary side (low voltage)
-A [Ratio Tap Changer](ratioTapChanger.md) can be associated to the side 2 or the side 3 of a three windings power transformer.
+- Side 2 is the secondary side (medium voltage)
+- Side 3 is the tertiary side (low voltage)
```
Comment (MioRtia) [inline docs/iidm/model/threeWindingsTransformer.md]: Is there a requirement for side 2 to be medium voltage or are side 2 and side 3 interchangeable?

File: docs/iidm/model/threeWindingsTransformer.md [hunk 3]
```diff
@@ -34,24 +45,19 @@ Three windings transformers can also have [current limits](currentLimits.md) def
 | B | double | S | yes | - | The nominal magnetizing susceptance specified at the voltage of the leg |
 | RatedU | double | kV | yes | - | The rated voltage |
 
-## Leg2or3
-`ThreeWindingsTransformer.Leg2or3` is a nested interface used to model the secondary or the tertiary side of a three windings
-power transformer.
-
-### Characteristics
+# Model
+Three windings transformer are modeled as three two windings transformers with the network bus at the end 1
```
[the remark this answers is not in the document]
Comment (zamarrenolm) [reply docs/iidm/model/threeWindingsTransformer.md]: instead of "really close" we have written "electrically equivalent".

File: docs/iidm/model/threeWindingsTransformer.md (hunk 1 above)
Comment (zamarrenolm) [reply docs/iidm/model/threeWindingsTransformer.md]: In IIDM there was no explicit check on the order of the legs. In the previous documentation, it was stated that side 1 should correspond to primary, and side 2 and 3 were interchangeable, but this was not enforced in the code.

We have written the documentation according to the current code. There is no check on the values of the rated or base voltages of each side.

In CGMES, there is a rule about the order of transformer ends (1-primary, 2-secondary, 3-tertiary), and this is respected when building the IIDM legs.

File: docs/iidm/model/threeWindingsTransformer.md (hunk 2 above)
Comment (zamarrenolm) [reply docs/iidm/model/threeWindingsTransformer.md]: see previous comment.

File: docs/iidm/model/threeWindingsTransformer.md (hunk 1 above)
Comment (MioRtia) [reply docs/iidm/model/threeWindingsTransformer.md]: Okay, I misunderstood this bit: as it was written, I thought it meant a t3w was connected to 3 voltage levels that were *usually* in the same substation but not always, this was why I was confused...

Maybe reword the sentence as below:
`A three windings transformer [...] to the same substation. We usually have:` 
`- side 1 as the primary side with the highest voltage`
`- side 2 as the secondary side with the medium voltage`
`- side 3 as the tertiary side with the lowest voltage`

Comment (zamarrenolm) [reply docs/iidm/model/threeWindingsTransformer.md]: done
